config/dependencies.yml

```diff
@@ -34,7 +34,7 @@ grpc:
     osx:
       deployment_target: '10.8'
   python:
-    version: '0.13.0'
+    version: '0.13.1'
   ruby:
     version: '0.13.0'
```

Comment:
it seems ruby-grpc is also updated to 0.13.1. Please update it as well.
https://rubygems.org/gems/grpc